src/lib/bootloader/grub2base.rb

```diff
@@ -93,8 +93,6 @@ def read
     end
 
     def write
-      super
```

Comment:
ah, you remove it. I think it is bad idea. If we have something to do always, then we can add it to base. I prefer keeping it empty and calling super here.

Reply:
Ok.